### PR TITLE
SA-12823: Include aria label for phone number

### PR DESCRIPTION
**Pull request by smeneni**
## Description
SA-12823: Include aria label for phone number

## Original issue(s)
department-of-veterans-affairs/va.gov-team [#SA-12823](https://jira.np.afsp.io/browse/SA-12823)

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/utils/labels.jsx [hunk 1]
```diff
@@ -193,13 +193,15 @@ export const directDepositWarning = (
           <b>or</b>
         </li>
         <li>
-          Call <a href="tel:8003331795">800-333-1795</a>
+          Call{' '}
+          <va-telephone contact="8003331795"></va-telephone>
```
Comment (nathanielbuck) [inline src/applications/edu-benefits/utils/labels.jsx:197]: Nice use of <va-telephone>, but you can make these self closing (`<va-telephone contact="8003331795" />`).

File: src/applications/edu-benefits/utils/labels.jsx [hunk 2]
```diff
@@ -193,13 +193,15 @@ export const directDepositWarning = (
...
         </li>
       </ul>
       If you choose not to enroll, you’ll need to call the Department of the
-      Treasury at <a href="tel:8882242950">888-224-2950</a> and speak to a
-      representative handling waiver requests. They’ll encourage you to
-      participate in EFT and address any questions or concerns you have.
+      Treasury at{' '}
+      <va-telephone contact="8882242950"></va-telephone>
```
Comment (nathanielbuck) [inline src/applications/edu-benefits/utils/labels.jsx:202]: Again, make this self-closing, please.

**Review by nathanielbuck — CHANGES_REQUESTED** [review]
Please make `<va-telephone>` elements self-closing.

File: src/applications/edu-benefits/utils/labels.jsx (hunk 1 above)
Comment (smeneni) [reply src/applications/edu-benefits/utils/labels.jsx]: resolved

File: src/applications/edu-benefits/utils/labels.jsx (hunk 2 above)
Comment (smeneni) [reply src/applications/edu-benefits/utils/labels.jsx]: resolved

**Review by nathanielbuck — APPROVED** [review]

**Pull request merged**